access/ip_access_list.go

```diff
@@ -0,0 +1,57 @@
+package access
+
+import "github.com/databrickslabs/databricks-terraform/common"
```

Comment:
move this file to same place as resource - if it's not re-used in other places, keep it in one file